src/content/en/fundamentals/instant-and-offline/get-started/wrapup-7.md

```diff
@@ -0,0 +1,532 @@
+project_path: /web/fundamentals/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+
+{# wf_updated_on: 2017-10-18 #}
+{# wf_published_on: 2017-10-18 #}
+
+# Wrap-up and Demo {: .page-title }
+
+{% include "web/_shared/contributors/dgash.html" %}
+
+
+We've now explored various ways to improve page load time. These techniques are "low-hanging 
+fruit" -- simple efforts that produce big performance wins. But talk is cheap; we have not 
+yet seen the techniques in action, so let's do that now.
+
+To demonstrate the techniques' effectiveness, we will run Lighthouse audits on different 
```

Comment:
The doc doesn't cover lighthouse results-- it uses other tools for testing performance.